subprojects/packagefiles/jbigkit/pbmtools/meson.build

```diff
@@ -0,0 +1,19 @@
+jbgtopbm = executable('jbgtopbm',
+    'jbgtopbm.c',
+    include_directories: '../libjbig',
+    link_with: libjbig)
```

Comment:
If you use `dependencies: libjbig_dep` it will handle both link_with and include_directories for you.